arches/app/media/js/viewmodels/map.js

```diff
@@ -479,6 +479,48 @@ define([
             });
         };
 
+        this.beforeMove = function(e) {
+            e.cancelDrop = (e.sourceParent!==e.targetParent);
+        };
+
+        this.reorderOverlays = function(e) {
+            const mapOrder = ko.observableArray(e.sourceParent());
+            const newOrder = self.createNewOverlayOrder(mapOrder);
+            self.sendNewOverlayOrder(newOrder)
+        };
+
+        this.keyDownHandler = function (context, e) {
```

Comment:
I think `var` can be replaced with `const` on the variables in this function except for `newIndex`; that should be `let`.